src/main/java/org/example/repository/OrderRepository.java

```diff
@@ -2,24 +2,95 @@
 
 import org.example.entity.Order;
 
-public class OrderRepository implements CrudRepository<Order> {
+import java.sql.*;
+
+public abstract class OrderRepository implements CrudRepository<Order> {
```

Comment:
Don't make it abstract. the fact that is's glowing red should tell you that you did something wrong. In this case you added additional methods to the interface and didn't implement them. Remove those methods and remove abstract keyword.